python/servo/command_base.py

```diff
@@ -522,13 +522,14 @@ def vs_dirs(self):
             'vcdir': vcinstalldir,
         }
 
-    def build_env(self, hosts_file_path=None, is_build=False, test_unit=False, features=None):
+    def build_env(self, hosts_file_path=None, is_build=False, test_unit=False):
         """Return an extended environment dictionary."""
         env = os.environ.copy()
 
-        if not features or "media-dummy" not in features:
+        if "media-gstreamer" in self.features:
```

Comment:
I think the WPT tests are crashing because the GStreamer env is not setup, since 'mach test-wpt' is not a 'build_like_command' (so self.features == []) and we no longer call `set_run_env` from [_test_wpt](https://github.com/servo/servo/blame/ae9843dcce0abf61818aaf4ee8668ac4299c28e9/python/servo/testing_commands.py#L386)

Reply:
Oh, you're right. We need to set these environment variables even if `self.features` hasn't been initialized.